extensions/amp-story/1.0/amp-story-page-attachment-header.css

```diff
@@ -24,11 +25,23 @@
 }
 
 .i-amphtml-story-page-attachment-close-button {
-  display: inline-block !important;
+  display: block !important;
   margin: 8px !important;
   height: 24px !important;
   width: 24px !important;
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 36 36" fill="rgba(0, 0, 0, 0.54)"><path d="M28.5 9.62L26.38 7.5 18 15.88 9.62 7.5 7.5 9.62 15.88 18 7.5 26.38l2.12 2.12L18 20.12l8.38 8.38 2.12-2.12L20.12 18z"/><path d="M0 0h36v36H0z" fill="none"/></svg>') !important;
   color: rgba(0, 0, 0, 0.87) !important;
   cursor: pointer !important;
 }
+
+.i-amphtml-story-page-attachment-title {
+  font-family: 'Roboto', sans-serif !important;
+  width: calc(100% - 80px) !important;
```

Comment:
This can't be done with flex?

Reply:
Not really: the close button on the left (or right if rtl) is 40px, so we also need to save 40px on the right to center the text. But there is no 3rd element for the flex layout to adjust, so we have to set an explicit width :/

![image](https://user-images.githubusercontent.com/1492044/56829210-20884b00-6831-11e9-8206-2af9a78a7f26.png)